### ISSUE TITLE
why the loss is going up??

### ISSUE THREAD

**Issue by zhongyuning** [computer-vision2022/PAMA#3]

**Comment by luoxuan-cs** [comment]
Please provide more information about the problem "loss going up". Meanwhile, did you modify the original code or the hyperparameters?

**Comment by zhongyuning** [comment]
> Please provide more information about the problem "loss going up". Meanwhile, did you modify the original code or the hyperparameters?

Thank you reply my question firstly!!
I change the code 
![image](https://user-images.githubusercontent.com/93465868/151109089-f0a11048-3d87-477a-8ee5-57075354848c.png)
, loss show  
![image](https://user-images.githubusercontent.com/93465868/151109130-9034c56c-42b6-47f3-b623-707d1a14d1c5.png)
,my question is why the loss is negative and goes up?

**Comment by luoxuan-cs** [comment]
Maybe you should run the original code and check if there are any bugs on your device & environment. Then you will be able to find the difference. Notice that the original code is designed to train with 2 GPUs, you should check the use of device.

**Comment by computer-vision2022** [comment]
Maybe you should run the original code and check if there are any bugs on your device & environment. Then you will be able to find the difference. Notice that the original code is designed to train with 2 GPUs, you should check the use of device.
